### ISSUE TITLE
git下面文件找不到

### ISSUE THREAD

**Issue by yanwaiwai** [oist/ecsu-dyadic_interaction#1]
git+https://gitlab.com/oist-ecsu/pyevolver@0.2.0，想问一下，这个对应下面的文件找不到是怎么回事？

**Comment by kercos** [comment]
English please 🙇

**Comment by yanwaiwai** [comment]
The corresponding file cannot be found under this url 。
https://gitlab.com/oist-ecsu/pyevolver

**Comment by kercos** [comment]
I see. Could you please write where you found this URL?

**Comment by yanwaiwai** [comment]
requirements.txt，Provided in line 25 of this file。

**Comment by yanwaiwai** [comment]
Hello, is there a solution?
Or am I misinterpreting this?

**Comment by kercos** [comment]
You are correct. Sorry for the confusion. This is because we have our repository on gitlab which is mirrored to github. 

However, only releases of the project connected to publications have github links in requirements. For instance, as explained in the [readme](https://github.com/oist/ecsu-dyadic_interaction#front-neurorobot-11-june-2021-paper) to replicate the results of Frontiers paper, you would need to check out v. [1.0.0](https://github.com/oist/ecsu-dyadic_interaction/tree/c1e3fdbd146eae2e7d17e0507b19c513c6ee5e92) which has correct github links in requirements.

For more recent versions, you would have to manually change the links to the github repository of pyevolver which you can find on [here](https://github.com/oist/ecsu-pyevolver/tags).

Please let me know if something is not clear.

**Comment by yanwaiwai** [comment]
Thank you for your help。
If there is anything clear, I will ask you again.
